### ISSUE TITLE
SinGAN implemenation

### ISSUE THREAD

**Issue by takuseno** [sony/nnabla-examples#112]
Hi, @TE-AkioHayakawa , @TE-TakuyaNarihira , @TE-KazukiYoshiyama san.

I've made SinGAN with NNabla. If you consider this to be good to this repository, I'll clean up codes and add more experiments.

https://github.com/takuseno/singan-nnabla/

https://arxiv.org/abs/1905.01164

**Comment by TE-TakuyaNarihira** [comment]
Yeah, that's really nice! Thanks for your contribution. Please give us a PR.
